frontend/components/TableContainer/TableContainer.tsx

```diff
@@ -127,7 +127,7 @@ interface ITableContainerProps<T = any> {
 
 const baseClass = "table-container";
 
-const DEFAULT_PAGE_SIZE = 20;
+export const DEFAULT_PAGE_SIZE = 20;
```

Comment:
This is to remove footer in some places if the page is page 0 and count is <= page size

Already on the future iteration list as:
RACHEL: We have a hideFooter property on the Table Container we’re constantly using if we’re on page zero and there’s no further pages, that should be built into TableContainer instead of trying to constantly define it as a prop.
EST: 2 pts